assgnment2/funcions.dart

```diff
@@ -0,0 +1,183 @@
+
```

Comment:
The files are not properly formatted (as per Dart guidelines). Make sure your code formatting is correct.

You can enable `Auto-Formatting on Save` feature for Dart on your code editor.